components/layout/Navbar.tsx

```diff
@@ -150,86 +167,84 @@ export function Navbar({}) {
           } z-40 w-full min-[1135px]:px-10 hidden min-[1135px]:flex items-center justify-between gap-6`}
         >
           <Link href={'/'}>
-              <h1 className="flex items-center">
-                <TinaIconSvg
-                  className={`${stuck ? 'w-8' : 'w-10'} h-auto fill-orange-500`}
-                />
-              </h1>
+            <h1 className="flex items-center">
+              <TinaIconSvg
+                className={`${stuck ? 'w-8' : 'w-10'} h-auto fill-orange-500`}
+              />
+            </h1>
           </Link>
           <nav className="flex-1 flex flex-wrap-reverse justify-end items-end min-[1135px]:items-center gap-2 min-[1135px]:gap-x-12">
             <ul className="flex gap-6 min-[1135px]:gap-8 min-[1135px]:gap-12 relative z-20">
-              {data.map((item) => {
-                if (item.href) {
-                  const { id, href, label, external } = item
-                  return (
-                    <li key={id} className={`group ${navLinkClasses}`}>
-                      <Link  href={href} className="py-2">
-                          {label}{' '}
-                          {external && (
-                            <BiLinkExternal
-                              className={`text-blue-200 group-hover:text-blue-400 inline`}
-                            />
-                          )}
-                      </Link>
-                    </li>
-                  )
-                } else {
-                  const { id, label } = item
-                  return (
-                    <li key={id} className={`group ${navLinkClasses} relative`}>
-                      <span className="py-2">{label}</span>
-                      <BiChevronRight
-                        className={`text-blue-200 group-hover:text-blue-400 group-hover:rotate-90 w-6 h-auto transition ease-out duration-300 transform`}
-                      />
-                      <ul className="absolute origin-top-right min-[1135px]:origin-top-left transition duration-300 ease-out opacity-0 group-hover:opacity-100 scale-95 group-hover:100 pointer-events-none group-hover:pointer-events-auto -translate-y-2 group-hover:translate-y-0 z-50 top-full -mt-0.5 right-0 min-[1135px]:right-auto min-[1135px]:-left-2 text-right min-[1135px]:text-left bg-white shadow-lg rounded-md px-4 py-3">
-                        {item.items &&
-                          item.items.map((child) => {
-                            const { id, href, label, external } = child
-                            return (
+              {navItems.map((item, index) =>
+                item._template === 'modalButton' ? (
```

Comment:
I've seen a few uses of the `modalButton` string in this file - Can we make it a const at the top?

Reply:
✅made it a const